lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.cpp

```diff
@@ -1363,9 +1375,51 @@ Status ProcessGDBRemote::DoResume() {
       }
     }
 
+    if (direction == RunDirection::eRunReverse && continue_packet_error) {
```

Comment:
Why do you care about continue_packet_error here?  That was being manipulated by the code that tried to go forward.  But you're going backwards here, so you shouldn't have gotten into any of that code, nor should you care what value the forward going code set this to.

Reply:
Removed `continue_packet_error` from here.